src/Approvals/SFA.DAS.Approvals.Api/Controllers/BulkUploadController.cs

```diff
@@ -94,5 +97,22 @@ public async Task<IActionResult> AddLog(BulkUploadAddLogRequest request)
 
             return Ok(result);
         }
+
+        [HttpPost]
+        [Route("UpdateLog")]
+        public async Task<IActionResult> UpdateLog(BulkUploadUpdateLogRequest request)
+        {
+            var result = await _mediator.Send(
+                new BulkUploadUpdateLogCommand
+                {
+                    LogId = request.LogId
+                });
+
+            if (result == null)
+            {
+                return NotFound();
+            }
+            return Ok(result);
+        }
```

Comment:
Again, we don't need this here yet, Can we remove it? It's also debatable that you would return anything other than OK()

Reply:
hi paul, removed all Update stuff